extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/build/BuildMetricsDevUIController.java

```diff
@@ -0,0 +1,327 @@
+package io.quarkus.devui.runtime.build;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.devui.runtime.build.BuildMetricsDevUIController.DependencyGraph.Link;
+import io.quarkus.devui.runtime.build.BuildMetricsDevUIController.DependencyGraph.Node;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class BuildMetricsDevUIController {
```

Comment:
This is more or less a copy of the old code, no real value here (except we return structures that can be serialized as JSON).
I marked the old code as deprecated so that we can drop it as soon as we remove the Dev UI 1.